assets/src/application/ui/styles/variables.scss

```diff
@@ -0,0 +1,73 @@
+// Variables
+//
+// Variables should follow the `$component-state-property-size` formula for
+// consistent naming. Ex: $nav-link-disabled-color and $modal-content-box-shadow-xs.
+
+// Color system
+
+$white: #fff !default;
+$gray-100: #f8f9fa !default;
+$gray-200: #e9ecef !default;
+$gray-300: #dee2e6 !default;
+$gray-400: #ced4da !default;
+$gray-500: #adb5bd !default;
+$gray-600: #6c757d !default;
+$gray-700: #495057 !default;
+$gray-800: #343a40 !default;
+$gray-900: #212529 !default;
+$black: #000 !default;
+
+// fusv-disable
+$grays: (
+	'100': $gray-100,
+	'200': $gray-200,
+	'300': $gray-300,
+	'400': $gray-400,
+	'500': $gray-500,
+	'600': $gray-600,
+	'700': $gray-700,
+	'800': $gray-800,
+	'900': $gray-900,
+) !default;
+// fusv-enable
+
+$blue: #0d6efd !default;
+$blue-5: #40a9ff !default;
+$blue-6: #1890ff !default;
```

Comment:
these variables are from here:
https://ant.design/docs/spec/colors

![image](https://user-images.githubusercontent.com/1477453/82840565-b894b000-9edb-11ea-93d8-0162d9fe3bcd.png)

Reply:
The reasons I've added these sass variables is because we can have safe guards, example below:


![Peek 2020-05-25 23-00](https://user-images.githubusercontent.com/1477453/82840578-c34f4500-9edb-11ea-8c37-9f34e5d5abdc.gif)

Reply:
the approach from: https://github.com/twbs/bootstrap/blob/master/scss/_variables.scss

Reply:
removed this. it was an experiment to see if build will fail if wrong color variable is applied;
***
currently, if I do write wrongly some CSS variable the build won't notice me in any, this is why I wrote that adding SASS variables will add safe guards in this regard

Reply:
gotcha... kk thnx for explanation